packages/gatsby-cli/src/reporter/redux/action-types.ts

```diff
@@ -0,0 +1,138 @@
+import { Actions, ActivityStatuses } from "../constants"
+import { IActivity } from "./reducer"
+import { Dispatch, AnyAction } from "redux"
+
+type dispatchAction = (dispatch: Dispatch<AnyAction>) => void
+
+export type actionsToEmitType = Array<
+  dispatchAction | { type: Actions; payload: Partial<IActivity> }
+>
+
+export interface IActionTypes {
+  setStatus(
+    status: string,
+    force?: boolean | undefined
+  ): (dispatch: Dispatch<AnyAction>) => void
+  createLog(arg0: {
+    text: string
+    level: string
+    duration: number
+    group?: string
+    code?: string
+    type?: string
+    filePath?: string
+    location?: string
+    docsUrl?: string
+    context?: string
+    statusText?: string
+    activity_uuid: string
+    activity_current: string
+    activity_total: number
+    activity_type: string
+    stack?: string
+  }): {
+    type: Actions
```

Comment:
this should be `Actions.SetStatus`

Reply:
Could you update all the `type` responses for all of these to be the proper enum value from `Actions`?